requirements.txt

```diff
@@ -10,3 +10,5 @@ requests
 beautifulsoup4==4.6.3
 semantic-version
 lxml
+pyelftools
```

Comment:
Is it OK? Because the error was `elftools`.

Reply:
Based on adjusted Pipfile, @KPostOffice was using pyelftools:

https://github.com/thoth-station/python/blob/dfb9c8d3d79b189967ef0f2eb1598c6581be4fec/Pipfile#L16

Reply:
Got it, thanks!